smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -173,14 +176,23 @@ private RuntimeClientPlugin(Builder builder) {
     /**
      * Gets a list of additional parameters to be supplied to the
      * resolve function. These parameters are to be supplied to resolve
-     * function as Nth(N &gt; 1) positional arguments. The list is empty if
-     * there are no additional parameters.
+     * function as second argument. The map is empty if there are
+     * no additional parameters.
      *
-     * @return Returns the optionally present list of parameters.
-     * @see #getResolveFunction()
+     * @param model Model the operation belongs to.
+     * @param service Service the operation belongs to.
+     * @param operation Operation to test against.
+     * @return Returns the optionally present map of parameters.
      */
-    public List<String> getAdditionalResolveFunctionParameters() {
-        return additionalResolveFunctionParameters;
+    public Map<String, Object> getResolveFunctionParameters(
```

Comment:
I'm not a big fan of this name. I feel like the old name is more descriptive, because it in fact only returns additional parameters other than the previous config.

Reply:
Reintroduced additional in a function name in [`482ea04` (#353)](https://github.com/awslabs/smithy-typescript/pull/353/commits/482ea044081dd2fe40e89ced5ea03bb030f46c25)